### PR TITLE
NXP-25549: Fix Picture.Resize not working with Document input

### PR DESCRIPTION
**Pull request by ThibArg**

### CODE REVIEW

File: nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/operation/PictureResize.java [hunk 1]
```diff
@@ -66,8 +67,14 @@ public Blob run(DocumentModel doc) {
         Blob pictureBlob = null;
         MultiviewPicture mvp = doc.getAdapter(MultiviewPicture.class);
         if (mvp != null) {
-            pictureBlob = mvp.getView(mvp.getOrigin()).getBlob();
-        } else {
+            // picture:origin is not always set.
+            PictureView pv = mvp.getView(mvp.getOrigin());
+            if(pv != null) {
```
Comment (efge) [inline nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/operation/PictureResize.java]: Add a space before `(`

File: nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/operation/PictureResize.java [hunk 2]
```diff
@@ -66,8 +67,14 @@ public Blob run(DocumentModel doc) {
...
+                pictureBlob = pv.getBlob();
+            }
+        }
+        
+        if(pictureBlob == null) {
```
Comment (efge) [inline nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/operation/PictureResize.java]: Add a space before `(`

File: nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/test/java/org/nuxeo/ecm/platform/picture/core/test/TestPictureResize.java [hunk 3]
```diff
@@ -90,4 +99,36 @@ public void testResizer() throws Exception {
...
+    
+    @Test
+    public void testResizerForDoc() throws Exception{
+        // TODO: NXP-27622
+        assumeFalse(SystemUtils.IS_OS_MAC);
+        
+        DocumentModel pictureDoc = session.createDocumentModel("/", "testpicture", "Picture");
+        Blob source = Blobs.createBlob(FileUtils.getResourceFileFromContext("images/test.jpg"), "image/jpeg");
+        pictureDoc.setPropertyValue("file:content", (Serializable) source);
+        pictureDoc = session.createDocument(pictureDoc);
```
Comment (efge) [inline nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/test/java/org/nuxeo/ecm/platform/picture/core/test/TestPictureResize.java]: Please add a comment in the test explaining what's specific about the setup that we want to test.

**Comment by atchertchian** [comment]
validated by https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-atchertchian-master/191/

**Pull request merged**